smart-contracts/test/testBridgeBank.js

```diff
@@ -169,6 +171,55 @@ describe("Test Bridge Bank", function () {
       );
       afterUserBalance.should.be.bignumber.equal(0);
     });
+
+    it("should allow a user to burn tokens twice from the bridge bank", async function () {
+      const BridgeTokenFactory = await ethers.getContractFactory("BridgeToken");
+      const bridgeToken = await BridgeTokenFactory.deploy("rowan", "rowan", 18, state.constants.denom.rowan);
+
+      const doubleAmount = Number(state.amount) * 2;
+
+      await bridgeToken.connect(operator).grantRole(state.constants.roles.minter, operator.address)
+      await bridgeToken.connect(operator).mint(userOne.address, doubleAmount);
+      await bridgeToken.connect(userOne).approve(state.bridgeBank.address, doubleAmount);
+      await state.bridgeBank.connect(owner).addExistingBridgeToken(bridgeToken.address);
+  
+      await state.bridgeBank.connect(userOne).burn(
+        state.sender,
+        bridgeToken.address,
+        state.amount
+      );
+
+      let afterUserBalance = Number(await bridgeToken.balanceOf(userOne.address));
+      afterUserBalance.should.be.bignumber.equal(state.amount);
+
+      // Do it again
+      await state.bridgeBank.connect(userOne).burn(
+        state.sender,
+        bridgeToken.address,
+        state.amount
+      );
+
+      afterUserBalance = Number(await bridgeToken.balanceOf(userOne.address));
+      afterUserBalance.should.be.bignumber.equal(0);
+    });
+
+    it("should allow a user to burn a token that doesn't have a denom", async function () {
```

Comment:
No. Should not be able to burn a token without a denom string.
*make sure erowan still works!